src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -619,9 +619,16 @@ sub check_dither {
     my $self = shift;
 
     my $dthr = shift;		  # Ref to array of hashes containing dither states
+    my $backstop = shift;
     my %dthr_cmd = (ON => 'ENAB',   # Translation from OR terminology to dither state term.
 		    OFF => 'DISA');
 
+    my %standard_y_dither = (20 => 1087.0,
```

Comment:
I was under the impression/of the opinion that flagging all but the first two as  "non-standard" would be fine for this PR and that the rest would belong more in #96